src/aur.c

```diff
@@ -385,6 +404,28 @@ static int json_integer (void * ctx, long long val)
 	return 1;
 }
 
+static int json_double (void * ctx, double val)
```

Comment:
I cannot tell off the top of my head - need to investigate that.
But all other tags are there...